client/mock/api/models/login_model.ts

```diff
@@ -0,0 +1,15 @@
+import type { AxiosResponse } from '../_common/axios_response';
+
+export type LoginRequestData = {
+	user: string;
+	password: string;
+};
+
+type LoginResponseData = object;
+
+// Specializing common Axios response to use response data type for its data field
+// This can be safely reused for every concrete response.
+// There is no chance 'data' field name will change in Axios
+export type LoginResponse = Omit<AxiosResponse, 'data'> & {
```

Comment:
I love ts magic like this 😄

Reply:
It's so ugly 😀 Stackoverflow said this is the best way.

Reply:
I think the js it generates in the end is much uglier so don't worry about it 😄